client/src/js/services/util.js

```diff
@@ -134,4 +134,9 @@ function UtilService($filter) {
     }
     return cleaned;
   };
+
+  // moment() provides the current date, similar to the new Date() API. This requests the difference between two dates
+  service.getMomentAge = function (date, duration) {
```

Comment:
This is a good idea.

Unfortunately, this change breaks the unit tests for Session, since it introduces a dependency on moment into `util`.  Could you `inject(moment)` into those tests so that they pass?